math/Postfix_Calculator.cpp

```diff
@@ -0,0 +1,170 @@
+
+#include<iostream>
+using namespace std;
+#include<stack>
+#include<cstring>
+
+//Declaration of the functions
+void infix_to_postfix(char *in, char *post);
+bool is_operand(char ch);
+int priority(char opr);
+int postfix_evaluation(char *post);
+int calculate(int op1, int op2, char opr);
+
+//cur = current
+
+void infix_to_postfix(char *in, char *post)
+{
+	int j=0;
+	stack<int> s;
+
+	for( int i = 0 ; in[i] != '\0' ; i++ )
+	{
+		//if the cur ele is an operand
+		if( is_operand(in[i]) )
+		{
+			//append it into the postfix string
+			post[j++] = in[i];
+		}
+		else
+		if( in[i] == '(')//if the current is an opening brace
+		{
+			//push it into the stack
+			s.push(in[i]);
+		}
+		else
+		if( in[i] == ')')//if the cur is a closing brace
+		{
+			//pop all ele'e from the stack one by one and append them into the
+			//postfix string till its corresponding opening brace do not occures
+			while( s.top() != '(' )
+			{
+				post[j++] = s.top();
+				s.pop();
+			}
+			//pop an opening brace from the stack
+			s.pop();
+		}
+		else//if the cur is an operator
+		{
+			//if stack is not empty && priority(topmost ele) >= priority(cur ele)
+			while( !s.empty() && priority(s.top()) >= priority(in[i]))
+			{
+				//pop ele from the stack and append it into the postfix string
+				post[j++] = s.top();
+				s.pop();
+			}
+
+			s.push(in[i]);
+		}
+
+	}
+
+	//pop all ele's one by one from the stack and append them into the postfix string
+
+	while( !s.empty())
+	{
+		post[j++] = s.top();
+		s.pop();
+	}
+
+	post[j] = '\0';
+}
+
+
+int postfix_evaluation(char *post)
+{
+	int result;
+	stack<int> s;
+
+	for( int i = 0 ; post[i] != '\0' ; i++ )
+	{
+		//if the cur ele is an operand
+		if( is_operand(post[i]))
```

Comment:
Please move brackets one line above:

```c++
if (var1 == var2) {
    // something here
} else
    // something here
}
```
Same in various other parts of the code.